### PR TITLE
Feature rebuild user interface

### PR DESCRIPTION
**Pull request by Szymiks**
Resolves: https://github.com/Code-Poets/sheetstorm/issues/103

Opened new PR with new code for user interface.

### CODE REVIEW

File: users/static/users/scripts/float_menu.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$(document).ready(function () {
+    $('#sidebarCollapse').on('click', function () {
+        $('#sidebar').toggleClass('active');
+    });
+});
```
Comment (kbeker) [inline users/static/users/scripts/float_menu.js]: Missing newline end of line

File: users/static/users/center_style.css [hunk 2]
```diff
@@ -1,3 +1,11 @@
+body {
+    background-color: #f2f2f2;
+    width: 100%;
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+}
+
+
```
Comment (kbeker) [inline users/static/users/center_style.css]: No needed line

File: users/static/users/scripts/float_menu.js (hunk 1 above)
Comment (Szymiks) [reply users/static/users/scripts/float_menu.js]: Done

File: users/static/users/center_style.css (hunk 2 above)
Comment (Szymiks) [reply users/static/users/center_style.css]: Done

**Pull request merged**